maintain_pim/common_issues/index.rst

```diff
@@ -114,27 +114,17 @@ To do so, create a ``my_index_configuration.yml`` file with the following conten
 
 Then, load the ``my_index_configuration.yml`` by adding it to the Symfony ``elasticsearch_index_configuration_files`` parameter that is present in the file ``parameters.yml`` or ``pim_parameters.yml``.
 
-For instance, if you have a *Community* edition:
+For instance:
 
 .. code-block:: yaml
 
     # parameters.yml
     parameters:
         # ...
+        # Files used as configuration for the Elasticsearch index
         elasticsearch_index_configuration_files:
-            - '%kernel.root_dir%/../vendor/akeneo/pim-community-dev/src/Akeneo/Pim/Enrichment/Bundle/Resources/elasticsearch/index_configuration.yml'
-            - '/path/to/my_index_configuration.yml'
-
-If you have the *Enterprise* edition:
-
-.. code-block:: yaml
-
-    # parameters.yml
-    parameters:
-        # ...
-        elasticsearch_index_configuration_files:
-            - '%kernel.root_dir%/../vendor/akeneo/pim-community-dev/src/Akeneo/Pim/Enrichment/Bundle/Resources/elasticsearch/index_configuration.yml'
```

Comment:
This file in v2.x contains mappings of pim_catalog_product:
![Screenshot from 2020-03-10 10-27-31](https://user-images.githubusercontent.com/34027529/76298468-c6ae3980-62b9-11ea-8044-ce60a3f97c6d.png)

Your fix is to show the settings... so I think it's incorrect
![Screenshot from 2020-03-10 10-28-03](https://user-images.githubusercontent.com/34027529/76298536-e34a7180-62b9-11ea-9c86-1f215f2e1a36.png)
.

Reply:
I may be wrong, but this PR is just about adding the custom ES config to the existing ones.
Settings AND mapping were already present in the configuration.